.eslintrc.js

```diff
@@ -2,78 +2,97 @@ module.exports = {
   root: true,
   env: {
     node: true,
-    browser: true,
+    browser: true
   },
   extends: [
     // Base extends
-    'eslint:recommended',
-    
+    "eslint:recommended",
+
     // Vue extends
-    'plugin:vue/base',
-    'plugin:vue/essential',
-    'plugin:vue/strongly-recommended',
-    'plugin:vue/recommended',
+    "plugin:vue/base",
+    "plugin:vue/essential",
+    "plugin:vue/strongly-recommended",
+    "plugin:vue/recommended",
+    "prettier",
+    "prettier/vue"
   ],
   rules: {
     // Disable console logs on production branch
-    'no-console': process.env.NODE_ENV === 'prod' ? 2 : 1,
-    'no-debugger': process.env.NODE_ENV === 'prod' ? 2 : 1,
-    
+    "no-console": process.env.NODE_ENV === "prod" ? 2 : 1,
+    "no-debugger": process.env.NODE_ENV === "prod" ? 2 : 1,
+    "no-unused-vars": "off",
```

Comment:
Ah looks like you did it here instead. Set it to warning - and move down to // Non-vue rules
I looked it up and I'd actually maybe suggest deactivating the option "args" totally, but not sure (i know in APIs its often seen where function arguments are declared but not used)

Reply:
i'll move this up to vue rules again because turns out it's actually a vue rule

Reply:
erhh it shouldn't be a vue rule - hence not prefixed with a `vue/`...